src/Mayaqua/Encrypt.c

```diff
@@ -3655,6 +3660,17 @@ void FreeCryptLibrary()
 	openssl_lock = NULL;
 //	RAND_Free_For_SoftEther();
 	OpenSSL_FreeLock();
+
+	FIPS_mode_set(0);

```

Comment:
is it safe to call when FIPS is not enabled?

Reply:
Yes.

Reply:
is SE capable of running in FIPS mode ? should we add that to travis-ci matrix ?

besides that, I think we can merge this PR

Reply:
I think we should add it to the Travis CI matrix, yes, as conformance to FIPS 140-2 is very important for a VPN software.